app/services/idv/steps/verify_base_step.rb

```diff
@@ -78,7 +73,8 @@ def should_use_aamva?(pii_from_doc)
       end
 
       def aamva_state?(pii_from_doc)
-        AAMVA_SUPPORTED_JURISDICTIONS.include?(pii_from_doc['state_id_jurisdiction'])
+        IdentityConfig.store.aamva_supported_jurisdictions.include?
+        (pii_from_doc['state_id_jurisdiction'])
```

Comment:
This is a bit difficult to read, particularly without any indentation. I might suggest one of the following:

```suggestion
        aamva_supported_jurisdictions.include?(pii_from_doc['state_id_jurisdiction'])
      end

      def aamva_supported_jurisdictions
        IdentityConfig.store.aamva_supported_jurisdictions
```

```suggestion
        IdentityConfig.store.aamva_supported_jurisdictions.
          include?(pii_from_doc['state_id_jurisdiction'])
```

```suggestion
        IdentityConfig.store.aamva_supported_jurisdictions.include?(
          pii_from_doc['state_id_jurisdiction'],
        )
```